test/lib/graphql/schema_comparator/diff/enum_test.rb

```diff
@@ -1,6 +1,6 @@
 require "test_helper"
 
-class GraphQL::SchemaComparator::Diff::DirectiveTest < Minitest::Test
+class GraphQL::SchemaComparator::Diff::EnumTest < Minitest::Test
```

Comment:
@eapache here's the answer

Reply:
wat

🤔  nice find (how?)

So because the class is the same name _and_ has the same methods (`setup` and `test_diff`) it was just completely blowing away the actual directive test when it was loaded.

Reply:
I figured either the filename or the class declaration was wrong, so I randomly opened up the enum test file to compare it and then noticed this 😂